src/App.tsx

```diff
@@ -8,27 +8,31 @@ import About from './components/AboutComponent/AboutComponent'
 import Contact from './components/ContactComponent/ContactComponent'
 import UndefinedRoute from './components/UndefinedRouteComponent/UndefinedRouteComponent'
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
-import { getCurrencyValues } from './services/CurrencyService'
-import { from } from 'rxjs'
+import { useCurrenciesFetch } from "./services/CurrencyService";
 
 const {Header, Content} = Layout
 
-const resource = getCurrencyValues()
-const $resourceObservable = from(resource)
-
-type Response = {
-    rates: ApiRate[];
-    date: string
+export interface CurrencyHistoryData {
```

Comment:
Shouldn't this be a type instead or interface? It's a common assumption to use interfaces for API requests / response and types for cross components and internal only data.

Reply:
Thank you Radek bro

Reply:
Will be fixed